assets/js/stripe.js

```diff
@@ -67,6 +70,25 @@ jQuery( function( $ ) {
 			stripe_cvc.mount( '#stripe-cvc-element' );
 		},
 
+		iterateCreditCardIcons: function() {
+			if ( cardIconsInterval ) {
+				clearInterval( cardIconsInterval );
+			}
+			const icons = $( '#payment .card-brand-icons img' );
+			if ( icons.length ) {
+				$( icons[ 0 ] ).addClass( 'icon-visible' );
+				let i = 0;
+				cardIconsInterval = setInterval( function() {
+					if ( i === icons.length ) {
+						i = 0;
+					}
```

Comment:
```suggestion
					i = i % icons.length;
```
This is slightly pedantic since I don't think it really affects performance or anything, but you could rewrite this expression as a one-liner with a modulus.

Reply:
Good idea! Added suggestion in d842c5b0aa818fb247f06204cae63420d7823147.